Makefile

```diff
@@ -68,3 +68,6 @@ endif
 # redirect all
 $(TARGETS):
 	$(MAKE) --directory=vmdk_plugin $@
+
+.DEFAULT:
```

Comment:
What is the role of this target given that we define the default target earlier in the file?